extensions/ql-vscode/test/vscode-tests/cli-integration/skeleton-query-wizard.test.ts

```diff
@@ -83,11 +83,12 @@ describe("SkeletonQueryWizard", () => {
     jest.spyOn(workspace, "workspaceFolders", "get").mockReturnValue([
       {
         name: `codespaces-codeql`,
-        uri: { fsPath: storagePath },
+        uri: { fsPath: storagePath, scheme: "file" },
       },
       {
         name: "/second/folder/path",
-        uri: { fsPath: storagePath },
+        uri: { fsPath: storagePath, scheme: "file" },
+        scheme: "file",
```

Comment:
Typo? Or do we need both `scheme` properties? 🤔

Reply:
Definitely a typo! Thanks!